jscomp/frontend/ast_exp_apply.ml

```diff
@@ -128,6 +128,12 @@ let app_exp_mapper (e : exp) (self : Bs_ast_mapper.mapper) (fn : exp)
                 pexp_loc = e.pexp_loc;
                 pexp_attributes = e.pexp_attributes;
               }
+          | Pexp_ident _ ->
```

Comment:
Not sure I understand this.
It does not seem only about attributes.
Apart from attributes, what else changes without this case?
Isn't this case handled somewhere below?

Reply:
I got your point. I think this commit change handles in more comprehensive cases. https://github.com/rescript-lang/rescript-compiler/pull/5585/commits/16a42b70d675f7b69f008ea659754c01e264fd52

Reply:
Indeed it's handled below in:
```ml
| _ -> Ast_compatible.app1 ~loc fn new_obj_arg
```

and can be handled as

```ml
{(Ast_compatible.app1 ~loc fn new_obj_arg) with pexp_attributes = e.pexp_attributes}
```

Reply:
> I got your point. I think this commit change handles in more comprehensive cases. [16a42b7](https://github.com/rescript-lang/rescript-compiler/commit/16a42b70d675f7b69f008ea659754c01e264fd52)

perfect

Reply:
It seems same, doesn't it?
```ocaml
Ast_compatible.app1 ~loc ~attrs:e.pexp_attributes fn new_obj_arg
```